Makefile

```diff
@@ -330,10 +330,10 @@ ${JERRY_BASE}/CMakeLists.txt: update
 -.PHONY: setup
 setup: ${JERRY_BASE}/CMakeLists.txt
 ifeq ($(ASHELL), ashell)
-ifeq ($(filter ide,$(MAKECMDGOALS)),ide)
-	@echo CONFIG_USB_CDC_ACM=n >> prj.conf
-else
+ifeq ($(filter ide,$(MAKECMDGOALS)),cli)
 	@echo CONFIG_USB_CDC_ACM=y >> prj.conf
+else
```

Comment:
One of these cases is not needed, the default is either y or n, then depending on the condition, you enable it or disable it

Reply:
Turns out neither is needed since they are provided by the .json files, so removing the whole logic block.